spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -206,7 +205,7 @@ private static File filesDirectory(Context context, String directoryType, String
     File directory;
     if (Build.VERSION.SDK_INT >= 21) {
       // Use external storage.
-      directory = new File(getExternalStorageDirectory(), "app_" + directoryType);
+      directory = new File(context.getExternalFilesDir(null), "app_" + directoryType);
```

Comment:
does this still work with `SpoonDeviceRunner`'s expectation of where the screenshots will be?  I don't have a Spoon rig set up at the moment or I'd just install locally and confirm.